View/ViewHandler.php

```diff
@@ -363,7 +363,12 @@ public function createRedirectResponse(View $view, $location, $format)
     public function renderTemplate(View $view, $format)
     {
         if (null === $this->templating) {
-            throw new \LogicException(sprintf('An instance of %s must be injected in %s to render templates.', EngineInterface::class, __CLASS__));
+            throw new \LogicException(sprintf(
```

Comment:
please update the constructor annotation `@param EngineInterface|Environment $templating`

Reply:
@Tobion Done.